Snippets/ABSDataBus/ABSDataBus_4/ABSDataBus_4.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net48</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <RootNamespace>ABSDataBus_5</RootNamespace>
```

Comment:
Is the root namespace needed?

Reply:
Probably not given the snippets project is for v5 and the namespace says v5 😂